build/images/Dockerfile.build.windows

```diff
@@ -27,6 +28,8 @@ COPY . /antrea
 
 RUN sh -c 'make windows-bin'
 
+FROM antrea/windows-ovs:${WIN_BUILD_OVS_TAG} as windows-ovs
```

Comment:
Since this image is including files from ovs image layer, we should make sure whenever we have new antrea-ovs image, we shld generate new antrea-windows image .